modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -126,7 +126,15 @@ const EditorManager: EditorManager = {
 
   baseURI: null as any,
   baseURL: null as any,
+
+  /**
+   * Object which contains required, and user-specific, default values for cloud-related options.
```

Comment:
Unfortunately, that's not what `defaultOptions` is. That's what we use it for on the cloud, but that's not what it is. It is a set of options that will be passed in by default to every `init` call, but the `init` call's settings / options etc. can override them.

Reply:
Thanks for the explanation! I wasn't sure where to find this out, so this was an uneducated placeholder. I've updated the description now, if you could please approve it.

Reply:
That still doesn't really explain how it's merged or that it's even be merged at all with the options passed to init. Maybe add something like this?
```
Object containing the options that will be passed by default to the `init` method upon each initialization of an editor. These defaults will be shallow merged with other options passed to `init`.
```

Reply:
Any better now?

Reply:
It's probably clearer if you keep it as two sentences. The reason is that the "that will be" has a slightly more ambiguous wording in the one-sentence version. 

> Object containing the default options that will be shallow merged with the options object passed to the `init` method upon each initialization of an editor.

It can be interpreted as it will contain *only* those default options that will be shallow merged, but other default options might be stored elsewhere. The two sentence version makes it a bit clearer that this will contain *all* the default options, and that all the default options will be passed through and shallow merged.

> Object containing the options that will be passed by default to the init method upon each initialization of an editor. These defaults will be shallow merged with other options passed to init.

But it's probably pretty subjective, so you might want to get another opinion.